SingularityClient/src/main/java/com/hubspot/singularity/client/SingularityClient.java

```diff
@@ -562,6 +566,26 @@ public SingularityState getState(Optional<Boolean> skipCache, Optional<Boolean>
     return Optional.of(response.getAs(SingularityTaskReconciliationStatistics.class));
   }
 
+  public SingularityClusterUtilization getClusterUtilization() {
```

Comment:
I would follow the format of the other get calls here. You'll see there is a `getSingle` method where the http call building and deserialization is done for you